actions/autosync/action.yml

```diff
@@ -32,7 +32,7 @@ inputs:
   skip_items:
     description: "Comma-separated glob patterns list of content by trestle name to skip during task execution. For example `profile_x,profile_y*,`."
     required: false
-  ssp_index_path:
+  ssp_index_file:
```

Comment:
Would the description need to change since this will be a file rather than a path? Possibly ->  "JSON file relative to the repository path where the ssp index is located. "